handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
`OPENSSL_NO_OCSP` means we do not support OpenSSL's OCSP API. If we did not define it, loads of projects would fail to build because they, naturally, do not use BoringSSL's API. Like with any other project, your options are not to support OCSP stapling with BoringSSL (in which case `OPENSSL_NO_OCSP`, what you would have been checking anyway, Just Works and you need no additional work) or to add BoringSSL-specific code that does not look at `OPENSSL_NO_OCSP`. BoringSSL-specific code is obviously needed anyway for BoringSSL-specific API, so I don't see why having to ignore a preprocessor symbol which happens to have a similar name as the feature matters.